### PR TITLE
Add a guide for request hedging

### PR DESCRIPTION
**Pull request by easwars**

### CODE REVIEW

**Comment by easwars** [comment]
@temawi : Would you be able to review this? Thanks.

File: content/en/docs/guides/request-hedging.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+---
+
+### Overview
+
+Hedging is one of two configurable retry policies supported by gRPC. With
+hedging, a gRPC client sends multiple copies of the same request to different
+backends and uses the first response it receives. Subsequently, the client
+cancels any outstanding requests and forwards the response to the application.
+
+![Basic hedging diagram](../../../../static/img/basic_hedge.svg)
```
Comment (temawi) [inline content/en/docs/guides/request-hedging.md:14]: The diagram doesn't seem to render, at least in the preview.

File: content/en/docs/guides/request-hedging.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+all sent out at the same time.
+- `nonFatalStatusCodes`: an optional list of grpc status codes. If any of hedged
+requests fails with a status code that is not present in this list, all
+outstanding requests are canceled and the response is returned to the
+application.
+
+#### Hedging policy
+
+When the application makes an RPC call that contains a `hedgingPolicy`
+configuration in the Service Cconfig, the original RPC is sent immediately, as
```
Comment (temawi) [inline content/en/docs/guides/request-hedging.md:56]: "Config"

File: content/en/docs/guides/request-hedging.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+with a standard non-hedged call. After `hedgingDelay`` has elapsed without a
+successful response, the second RPC will be issued. If neither RPC has received
+a response after `hedgingDelay`` has elapsed again, a third RPC is sent, and so
```
Comment (temawi) [inline content/en/docs/guides/request-hedging.md:59]: extra `

**Review by temawi — COMMENTED** [review]
Nice guide, thanks! Left a few minor comments.

File: content/en/docs/guides/request-hedging.md (hunk 3 above)
Comment (easwars) [reply content/en/docs/guides/request-hedging.md:59]: Done.

**Review by easwars — COMMENTED** [review]
Thanks for the review.

File: content/en/docs/guides/request-hedging.md (hunk 1 above)
Comment (temawi) [reply content/en/docs/guides/request-hedging.md:14]: Click on "Details" for the "netlify/grpc-io/deploy-preview" GitHub check and you will get a preview of the whole site with your change. The image does not show up there. Maybe a path issue?

Comment (easwars) [reply content/en/docs/guides/request-hedging.md]: Renders now. Thanks. PTAL.

**Comment by temawi** [comment]
LGTM, thanks!

**Review by temawi — APPROVED** [review]

**Comment by easwars** [comment]
Thanks for the review @temawi

**Pull request merged**